views/js/qtiCommonRenderer/renderers/interactions/pci/common.js

```diff
@@ -58,7 +58,7 @@ define([
                pci.initialize(id, containerHelper.get(interaction).children().get(0), properties, pciAssetManager);
            },
            destroy : function destroy(interaction){
-               instanciator.getPci(interaction).destroy();
+               return instanciator.getPci(interaction).destroy();
```

Comment:
Could you please add the method comment (we always try to improve the part we modify)